### DISCUSSION TITLE
AXIS D4100-E - API Lights

### DISCUSSION THREAD

**Discussion by Bausera** [AxisCommunications/discussions#167] [VAPIX] [answered]
Hello everyone,

I need to trigger the Axis stroble lights through an API, but I didn't find any commands in the Axis libraries (https://www.axis.com/vapix-library/)

Tks.

**Comment by mattias-kindborg-at-work** [comment, marked as answer]
Which product are we talking about?

**Reply by Bausera** [reply]
AXIS D4100-E - https://www.axis.com/pt-br/products/axis-d4100-e-network-strobe-siren

**Reply by mattias-kindborg-at-work** [reply]
Would [Siren and light in VAPIX](https://www.axis.com/vapix-library/subjects/t10175981/section/t10148887/display) help you?

**Reply by Bausera** [reply]
Yes, thank you Mattias :D
